src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -108,22 +113,24 @@ const Packages = ({ defaultArch, ...props }) => {
 
     // call api to list available packages
     const handlePackagesAvailableSearch = async () => {
+        setFilterAvailable(packagesSearchName);
+
         const packageList = await getAllPackages();
         if (packageList) {
             const packagesAvailableFiltered = filterPackagesAvailable(packageList);
-            sortPackages(packagesAvailableFiltered);
-            setPackagesAvailableFound(true);
+            setPackagesAvailableSorted(packagesAvailableFiltered, packagesSearchName);
```

Comment:
This is why `setPackagesAvailableSorted()` needs an optional `filter` parameter, we need to use `packagesSearchName` here as the default `filterAvailable` would be the value from the previous render and therefore stale.